hca/util/__init__.py

```diff
@@ -131,11 +131,41 @@ def increment(self, *args, **kwargs):
 
 
 class _ClientMethodFactory(object):
-
     def __init__(self, client, parameters, path_parameters, http_method, method_name, method_data, body_props):
         self.__dict__.update(locals())
         self._context_manager_response = None
 
+    def request_with_retries_on_post_search(self, session, url, query, json_input, stream, headers):
```

Comment:
Is this method used in `_PaginatingClientMethodFactory`?

Reply:
It is.  `_PaginatingClientMethodFactory` calls `_request` which calls `request_with_retries_on_post_search`.